torch/csrc/autograd/variable.h

```diff
@@ -164,20 +164,20 @@ struct VariableViewImpl : public VariableImpl {
   int attr_version;
 };
 
-inline Variable make_variable(at::Tensor data) {
-  return Variable(new VariableImpl(std::move(data)), false);
+inline Variable make_variable(at::Tensor data, VarFlags flags=DEFAULT_FLAGS,
+                              int output_nr=0, std::shared_ptr<Function> grad_fn=nullptr) {
+  return Variable(new VariableImpl(std::move(data), flags, output_nr, std::move(grad_fn)), false);
 }
 
-inline Variable make_variable(at::Tensor data, std::shared_ptr<Function> grad_fn) {
-  return Variable(new VariableImpl(std::move(data), std::move(grad_fn)), false);
-}
+Variable make_variable(at::Tensor data, std::shared_ptr<Function> grad_fn);
 
 inline Variable make_variable(at::Tensor data, bool requires_grad, bool is_volatile=false) {
-  return Variable(new VariableImpl(std::move(data), requires_grad, is_volatile), false);
+  return make_variable(std::move(data), VarFlags(requires_grad, is_volatile));
```

Comment:
this overload is a bit weird now, as everything else uses `VarFlags`. but I don't mind having it

Reply:
Yeah, we should clean this up soon